packages/sensors_plus/sensors_plus/ios/sensors_plus.podspec

```diff
@@ -14,9 +15,10 @@ Flutter plugin to access the accelerometer, gyroscope, and magnetometer sensors.
   s.source           = { :http => 'https://github.com/fluttercommunity/plus_plugins/tree/main/packages/sensors_plus' }
   s.documentation_url = 'https://pub.dev/packages/sensors_plus'
   s.source_files = 'Classes/**/*'
-  s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
-
   s.platform = :ios, '11.0'
-  s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }
+
+  # Flutter.framework does not contain a i386 slice.
+  s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES', 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'i386' }
```

Comment:
Why is this change needed? Previously, there were no need for this to build for simulator.

Reply:
I don't know. Since swift language was mirgated to, the whole ios project have been regenerated by the latest version of flutter SDK. And this is the default setting.

Reply:
Understood. Thought it was added manually.